site/pages/submit-talk.en.rst

```diff
@@ -1,10 +1,10 @@
 .. title: Submit Talk Proposal
 .. slug: submit-talk
 .. date: 2018-03-16 19:22:54 UTC+07:00
-.. tags: 
-.. category: 
-.. link: 
-.. description: 
```

Comment:
They're generated with trailing space by Nikola, cleaning that up for all pages will be a losing battle  .. 
In this case I think trailing space makes sense anyway,  if you add anything you'll have to add the space anyway..

Reply:
Oh I see... I did not really care,  my editor is configured to clean trailing blanks when saving...